Eat_and_Greet/src/comps/CircularButton/style.tsx

```diff
@@ -2,14 +2,26 @@ import { StyleSheet } from 'react-native';
 
 const CircularButtonStyles = StyleSheet.create({
     button: {
-        borderRadius: 50,
-        backgroundColor: '#E5E0FF', 
+        borderRadius: 25,
+        backgroundColor: '#8EA7E9', 
         justifyContent: 'center',
         position: 'relative',
         alignItems: 'center',
+        alignSelf:'flex-start',
+        marginLeft: 20,
+        marginTop: 30,
+        marginRight: 20,
         shadowOffset: { width: 2, height: 2 }, 
         shadowColor: '#000',
         shadowOpacity: 0.5, 
+      },
+      ButtonText:{
```

Comment:
Is this not in the general Style sheet? Consider making the fontFamily in the style sheet and others here (Does it not inherit align and justify content from parents?)

Reply:
needed to change the size and some other information so added that in.